content/en/continuous_integration/intelligent_test_runner/how_it_works.md

```diff
@@ -0,0 +1,61 @@
+---
+title: How It Works
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/streamline-ci-testing-with-datadog-intelligent-test-runner/"
+    tag: "Blog"
+    text: "Streamline CI testing with Datadog Intelligent Test Runner"
+  - link: "https://www.datadoghq.com/blog/monitor-ci-pipelines/"
+    tag: "Blog"
+    text: "Monitor all your CI pipelines with Datadog"
+---
+
+## What is Intelligent Test Runner?
+
+Intelligent Test Runner is Datadog's test impact analysis solution. Test impact analysis is a technique that has been gaining popularity over the past few decades. However, it's typically hard and time-consuming to implement. Intelligent Test Runner simplifies this complexity with an easy setup process.
+
+Test impact analysis maps each test to the set of code files in your repository that your test uses. Its goal is to skip tests that are not affected by the code changes that are being introduced, in order to reduce the time taken by the testing phase of your CI.
+
+An extreme example is a Pull Request that only changes a typo on a README file. For that PR, running all tests doesn't provide much value. On the contrary, flaky tests might make your CI fail, forcing you to retry the pipeline, potentially multiple times, before merging. This is a waste of both developer and CI time. With Intelligent Test Runner, a PR changing a README file would skip all tests.
+
+## What sets it apart
+
+Some test selection solutions try to make up for the lack of code coverage information by trying to infer in a probabilistic fashion which tests are relevant through the use of Machine Learning. This results in a test selection mechanism that can skip tests that were relevant, and cause build failures in your default branch.
+
+Other test impact analysis solutions are based off code coverage but are hard to set up. Additionally, they only look at the diff in your latest commit to evaluate which tests to run. This means that you need to guarantee that all your commits run through CI. Even if you do, this doesn't play well with GitHub's Pull Requests, which only take into account the CI status of the latest commit to allow merging the PR.
```

Comment:
re: warning - change last sentence:

"Even if they do, there still may be issues related to GitHub's Pull Requests. When it comes to merging the PR, only the CI status of the latest commit is taken into account."